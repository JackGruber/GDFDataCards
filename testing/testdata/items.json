{
    "id": "M4FVYNIo7I5A",
    "gameSystem": "gff",
    "armyId": "7oi8zeiqfamiur21",
    "armyIds": [
        "7oi8zeiqfamiur21"
    ],
    "armyFaction": null,
    "armyName": "Blessed Sisters",
    "modified": "2024-10-29T17:41:57.898Z",
    "saveVersion": 3,
    "favourite": false,
    "armyVersions": [
        {
            "version": "3.4.1",
            "armyId": "7oi8zeiqfamiur21"
        }
    ],
    "listPoints": 135,
    "list": {
        "id": "M4FVYNIo7I5A",
        "key": "e_5Es-hjcVZKZVeD8e4Al",
        "name": "Blessed Sisters",
        "description": "",
        "pointsLimit": 300,
        "campaignMode": false,
        "narrativeMode": false,
        "forceOrg": true,
        "gameSystem": "gff",
        "isCloud": false,
        "modified": "2024-10-29T17:41:57.898Z",
        "cloudModified": null,
        "activationCount": 2,
        "modelCount": 2,
        "units": [
            {
                "id": "2TXWi3B",
                "armyId": "7oi8zeiqfamiur21",
                "selectionId": "K-c1E",
                "selectedUpgrades": [
                    {
                        "instanceId": "dVNhF0sbx",
                        "upgradeId": "jDPiWW0",
                        "optionId": "se1bXdC"
                    },
                    {
                        "instanceId": "yJfR_XD-j",
                        "upgradeId": "QiaDUvj",
                        "optionId": "4nJa1"
                    },
                    {
                        "instanceId": "fNUmwNbmQ",
                        "upgradeId": "9LBlxtK",
                        "optionId": "5qZXT6w"
                    },
                    {
                        "instanceId": "DIEVeqhSN",
                        "upgradeId": "5z9UHa0",
                        "optionId": "OqMP9"
                    }
                ],
                "combined": false,
                "joinToUnit": null,
                "xp": 0,
                "traits": [],
                "notes": null
            },
            {
                "id": "2TXWi3B",
                "armyId": "7oi8zeiqfamiur21",
                "selectionId": "zEHLc",
                "selectedUpgrades": [],
                "combined": false,
                "joinToUnit": null,
                "xp": 0,
                "traits": [],
                "notes": null
            }
        ]
    }
}