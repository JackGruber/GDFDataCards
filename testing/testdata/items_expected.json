{
    "armyId": "7oi8zeiqfamiur21",
    "gameSystem": "gff",
    "gameSystemId": 3,
    "armyVersions": [
        {
            "version": "3.4.1",
            "armyId": "7oi8zeiqfamiur21"
        }
    ],
    "armyName": "Blessed Sisters",
    "listPoints": 135,
    "listName": "Blessed Sisters",
    "units": [
        {
            "type": "High Sister",
            "name": "High Sister",
            "armyId": "7oi8zeiqfamiur21",
            "id": "2TXWi3B",
            "cost": 35,
            "defense": 4,
            "quality": 4,
            "upgrades": [
                "A1",
                "B1",
                "C1"
            ],
            "size": 1,
            "notes": null,
            "weapons": [
                {
                    "attacks": 2,
                    "count": 1,
                    "name": "Master Heavy Rifle",
                    "range": 24,
                    "specialRules": [],
                    "ap": 1
                },
                {
                    "attacks": 1,
                    "count": 1,
                    "name": "Heavy Mace",
                    "specialRules": [
                        {
                            "id": "w_vX0mi58KKt",
                            "name": "Blast",
                            "type": "ArmyBookRule",
                            "rating": 3,
                            "label": "Blast(3)"
                        }
                    ],
                    "ap": 2
                }
            ],
            "equipment": [
                {
                    "name": "Witch",
                    "specialRules": [
                        {
                            "id": "47CAqf7ySlfc",
                            "name": "Caster",
                            "type": "ArmyBookRule",
                            "rating": 2,
                            "label": "Caster(2)"
                        }
                    ]
                },
                {
                    "name": "Jetpack",
                    "specialRules": [
                        {
                            "id": "6mJw5IdqSqNC",
                            "name": "Ambush",
                            "type": "ArmyBookRule",
                            "label": "Ambush"
                        },
                        {
                            "id": "-QJdqJEKjIpE",
                            "name": "Flying",
                            "type": "ArmyBookRule",
                            "label": "Flying"
                        }
                    ]
                }
            ],
            "rules": [
                {
                    "id": "j_GPMzrugrCj",
                    "name": "Devout",
                    "additional": false,
                    "label": "Devout"
                },
                {
                    "id": "T08VkBuVmNZ_",
                    "name": "Hero",
                    "label": "Hero"
                },
                {
                    "id": "a0YtInGiUDd6",
                    "name": "Tough",
                    "rating": 3,
                    "label": "Tough(3)"
                },
                {
                    "id": "6mJw5IdqSqNC",
                    "name": "Ambush",
                    "type": "ArmyBookRule",
                    "label": "Ambush"
                },
                {
                    "id": "-QJdqJEKjIpE",
                    "name": "Flying",
                    "type": "ArmyBookRule",
                    "label": "Flying"
                }
            ],
            "upgradeCost": [
                20,
                25,
                10,
                10
            ]
        },
        {
            "type": "High Sister",
            "name": "High Sister",
            "armyId": "7oi8zeiqfamiur21",
            "id": "2TXWi3B",
            "cost": 35,
            "defense": 4,
            "quality": 4,
            "upgrades": [
                "A1",
                "B1",
                "C1"
            ],
            "size": 1,
            "notes": null,
            "weapons": [
                {
                    "attacks": 2,
                    "count": 1,
                    "name": "CCW",
                    "specialRules": []
                }
            ],
            "equipment": [
                {
                    "name": "Combat Shield",
                    "specialRules": [
                        {
                            "id": "wsqfB0fq69eG",
                            "name": "Shield Wall",
                            "type": "ArmyBookRule",
                            "label": "Shield Wall"
                        }
                    ]
                }
            ],
            "rules": [
                {
                    "id": "j_GPMzrugrCj",
                    "name": "Devout",
                    "additional": false,
                    "label": "Devout"
                },
                {
                    "id": "T08VkBuVmNZ_",
                    "name": "Hero",
                    "label": "Hero"
                },
                {
                    "id": "a0YtInGiUDd6",
                    "name": "Tough",
                    "rating": 3,
                    "label": "Tough(3)"
                }
            ]
        }
    ]
}